variables.tf

```diff
@@ -72,19 +79,74 @@ variable "cloud_connection_speed" {
 #####################################################
 
 variable "tags" {
-  description = "List of Tag names for IBM Cloud PowerVS service"
+  description = "List of tag names for the IBM Cloud PowerVS service"
   type        = list(string)
   default     = null
 }
 
 variable "cloud_connection_gr" {
-  description = "Enable global routing for this cloud connection.Can be specified when creating new connection"
+  description = "Whether to enable global routing for this IBM Cloud connection. You can specify thia value when you create a connection."
   type        = bool
   default     = null
 }
 
 variable "cloud_connection_metered" {
-  description = "Enable metered for this cloud connection. Can be specified when creating new connection"
+  description = "Whether to enable metering for this IBM Cloud connection. You can specify thia value when you create a connection."
   type        = bool
   default     = null
 }
+
+variable "access_host_or_ip" {
+  description = "The public IP address for the jump or Bastion server. The address is used to reach the target or server_host IP address and to configure the DNS, NTP, NFS, and Squid proxy services."
```

Comment:
This could be the IP of any jump box reachable from the machine running the terraform logic. Could be a private IP in the scenario where this code is running from a vsi within the VPC) - although there is probably no need for a jump box in that scenario in theory (but code assumes there is one in practice)

Reply:
It could be any IP that is reachable from TF engine and allows access to the environment. Practically, this is public IP.